ubi-ruby-fips/Dockerfile

```diff
@@ -0,0 +1,20 @@
+ARG RUBY_BUILDER_TAG
+ARG BUNDLER_VERSION=2.1.4
+
+# Ruby Builder
+FROM ubi-ruby-builder:$RUBY_BUILDER_TAG as ubi-ruby-builder
+
```

Comment:
this is much more lightweight than the other image Dockerfiles; is there a reason not to copy files
```
FROM openssl-builder:$OPENSSL_BUILDER_TAG as OpenSSL-builder
FROM postgres-client-builder:$PG_BUILDER_TAG as postgres-client-builder
```

the same way the other images do?

Reply:
We don't need to build it separately because it comes FIPS on UBI 8 out of the box.